db/migrate/20220226134505_add_events_to_rules.rb

```diff
@@ -0,0 +1,5 @@
+class AddEventsToRules < ActiveRecord::Migration[6.1]
+  def change
+    add_column :rules, :events, :text
```

Comment:
Wait, if all you want is JSON instead of ruby marshalled objects, shouldn't this be a JSON column? This would allow us to avoid having to compare to `nil` and would give use automatic parse/dump?

Reply:
>  shouldn't this be a JSON column?

Yes

> This would allow us to avoid having to compare to nil

Only if we add the default value in mysql schema which I don't really agree with.

> would give use automatic parse/dump?

No. The JSON structure(as for text) does not know the object whcih needs to be re-hydrated. This is why vanilla rails serialization marshal dump the object data and class